deps/rabbitmq_cli/CONTRIBUTING.md

```diff
@@ -120,6 +120,15 @@ functions in Mix to start a new distributed node and connect it to the RabbitMQ
 server. It complains because we don't actually use the context dictionary, but
 it's fine otherwise.
 
+### Running Modified CLIs
+
+To execute the modified CLIs (to test against an actual broker), follow these steps:
+
+```
+make -C ../.. dist # make dist in the main rabbitmq-server repository folder
+../../sbin/rabbitmqctl # run CLIs from the main repository's sbin folder
```

Comment:
`make dist` can be executed in any RabbitMQ components, like `deps/rabbitmq_stomp`. The `sbin` directory appears where `make dist` was executed. It's a lot faster than running the same command at the root of the monorepository because it doesn't try to build all plugins.

Reply:
I've tried that and it doesn't work for me:

```
cd rabbitmq-server
make distclean
cd deps/rabbitmq_cli
make dist
./sbin/rabbitmqctl list_feature_flags
Error: {:unable_to_load_rabbit, {'no such file or directory', 'rabbit.app'}}

./sbin/rabbitmqctl
** (UndefinedFunctionError) function :rabbit_plugins.read_enabled/1 is undefined (module :rabbit_plugins is not available)
    :rabbit_plugins.read_enabled('/var/folders/r5/v5nlv8h9201510rkw_kdyg040000gn/T/rabbitmq-test-instances/rabbit@mkuratczyk-a01/enabled_plugins')
    (rabbitmqctl 3.8.0-dev) lib/rabbitmq/cli/core/command_modules.ex:60: RabbitMQ.CLI.Core.CommandModules.plugin_modules/1
    (rabbitmqctl 3.8.0-dev) lib/rabbitmq/cli/core/command_modules.ex:48: RabbitMQ.CLI.Core.CommandModules.load_commands/2
    (rabbitmqctl 3.8.0-dev) lib/rabbitmq/cli/core/command_modules.ex:33: RabbitMQ.CLI.Core.CommandModules.load/1
    (rabbitmqctl 3.8.0-dev) lib/rabbitmq/cli/ctl/commands/help_command.ex:212: RabbitMQ.CLI.Ctl.Commands.HelpCommand.commands_description/0
    (rabbitmqctl 3.8.0-dev) lib/rabbitmq/cli/ctl/commands/help_command.ex:92: RabbitMQ.CLI.Ctl.Commands.HelpCommand.all_usage/1
    (rabbitmqctl 3.8.0-dev) lib/rabbitmqctl.ex:40: RabbitMQCtl.exec_command/2
    (rabbitmqctl 3.8.0-dev) lib/rabbitmqctl.ex:31: RabbitMQCtl.main/1
```

Did I miss something?

Reply:
Sorry, by "RabbitMQ components" here, I meant, the broker or any plugins.

Reply:
The point is that an `sbin/` directory with CLI commands will be created in any subdirectory where you run e.g. `gmake run-broker` from. I'd say we can recommend running it from root, to have a well-known option.

There may be need to override `ERL_LIBS` to point to the `plugins` directory of a node built from source to make sure that commands provided by plugins are discovered.